python/README.md

```diff
@@ -54,10 +69,18 @@ The program will first randomly generate a number unknown to the user. The user
 ### TBD
 
 # Resources
+* [Automate the boring stuff](https://automatetheboringstuff.com/)
+
 * [Python 3 documentation tutorial](https://docs.python.org/3/tutorial/)
 
 * [The Coder's Apprentice](http://www.spronck.net/pythonbook/index.xhtml)
 
 * [PEP 8 - Style Guide for Python Code](https://www.python.org/dev/peps/pep-0008/)
 
 * [CS50's Introduction to Computer Science](https://www.edx.org/course/cs50s-introduction-to-computer-science)
+
+* [Python for Non-Programmers](https://wiki.python.org/moin/BeginnersGuide/NonProgrammers)
+
+* [How to think like a Computer Scientist](http://interactivepython.org/runestone/static/thinkcspy/index.html)
+
+* [Real Pyton](https://realpython.com/)
```

Comment:
s/Pyton/Python

Reply:
I don't get this

Reply:
I meant, `Real Pyton` should be `Real Python`

Reply:
Fixed